celery_once/backends/__init__.py

```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+
+from redis import Redis
```

Comment:
I think this needs a `from __future__ import absolute_import` or written as `from .redis import Redis`.